spec/spec_helper.rb

```diff
@@ -48,6 +48,7 @@
 
   config.include Capybara::DSL, :type => :request
   config.include Delorean
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
I think this is one of those things that makes it easier for us devs but harder for anyone coming to the codebase for the first time so I'm going to edit it out during the rebase/merge - sorry!